gatsby-config.js

```diff
@@ -8,6 +8,7 @@ const getAgentName = require('./src/utils/getAgentName');
 
 const siteUrl = 'https://docs.newrelic.com';
 const dataDictionaryPath = `${__dirname}/src/data-dictionary`;
+const additionalLocales = ['jp'];
```

Comment:
This probably doesn't need to be done now, but would it be worth adding a config variable to hold all the locales? That way when we add another we don't need to go through a bunch of plugins.

Reply:
Yes! We had discussed moving this into some sort of configuration file that also included locale codes in different formats (`ja-JP` for example). Out of scope of this PR, but certainly something we should consider.